packages/@lightningjs/ui-components/src/mixins/withThemeStyles/index.js

```diff
@@ -456,6 +460,35 @@ export default function withThemeStyles(Base, mixinStyle) {
       this.queueThemeUpdate();
     }
 
+    replaceAliasValues(value) {
```

Comment:
Didn't really think about it at the time, do you all have a preference for if this is a "private" (underscored) method or not?

Reply:
Yeah, prob could be underscored.

Reply:
+1 for private

Reply:
updated!